test/test-remote/looker/src/logs/index.ts

```diff
@@ -364,30 +338,30 @@ export class LogStreamFragmentPushRequest {
 
       const labelsEncodedAsString = logqlLabelString(fragment.labels);
 
-      // Create individual protobuf entries, and build up a checksum from the
+      // Create individual protobuf samples, and build up a checksum from the
       // textual content of all log messages.
-      const pbentries = [];
-      for (const entry of fragment.getEntries()) {
-        pbentries.push(
+      const pbsamples = [];
+      for (const entry of fragment.getSamples()) {
+        pbsamples.push(
           pbTypeEntry.create({
             timestamp: pbTypeTimestamp.create(entry.time),
             // Note(JP): it is a bit unclear which text encoding is actually
             // applied here. The protobuf definition has type `string`:
             // https://github.com/grafana/loki/blob/14b2c093c19e17103e564b0aa6af0cf16ff0e5bc/pkg/logproto/types.go#L20
             // the protobufjs library has to apply _some_ text encoding, and it's
             // likely to be UTF-8.
-            line: entry.text
+            line: entry.value
           })
         );
         // Update log text hash with the UTF-8-encoded version of the text.
         // From docs: "If encoding is not provided, and the data is a string,
         // an encoding of 'utf8' is enforced"
-        logTextHash.update(entry.text);
+        logTextHash.update(entry.value);
       }
 
       const stream = pbTypeStream.create({
         labels: labelsEncodedAsString,
-        entries: pbentries
+        samples: pbsamples
```

Comment:
:fire:  oh man I was debugging for the last hour why things fell apart.

This here was a bad search/replace. The fact that this compiles is sad!

`entries:` _must_ stay as-is (on the left-hand side), that's part of the protobuf message contract.